stackslib/src/chainstate/stacks/db/mod.rs

```diff
@@ -643,16 +643,8 @@ impl<'a> ChainstateTx<'a> {
         events: &[StacksTransactionReceipt],
     ) {
         if *TRANSACTION_LOG {
```

Comment:
I would just get rid of this variable entirely I think

Reply:
Agreed, but also add a clear message about this change to the changelog.